### PR TITLE
Added a plot_cbvs function and added errors to correct function

### PR DESCRIPTION
**Pull request by christinahedges**
I had to change the CBV corrector slightly for a tutorial. I changed the following:

* Moved how we obtain the CBVs to its own function `_get_cbv_data`
* Added the errors back into the `LightCurve` that the corrector returns (Although we think it shouldn't do this in future see #46 )
* Added a `plot_cbvs` function. This is useful for the tutorial.

### CODE REVIEW

File: lightkurve/correctors.py [hunk 1]
```diff
@@ -209,6 +216,37 @@ def get_cbv_url(self):
...
+        ax : matplotlib.pyplot.Axes.AxesSubplot
+            Matplotlib axis object
+        '''
+        if ax is None:
+            _, ax = plt.subplots(1)
+        cbv_array, time = self._get_cbv_data(cbvs)
+        for idx, cbv in enumerate(cbv_array):
+            ax.plot(time, cbv+idx/10., label='{}'.format(idx + 1))
+        ax.set_yticks([])
+        ax.set_xlabel('Time (MJD)')
```
Comment (barentsen) [inline lightkurve/correctors.py]: Are you sure this is MJD, not BKJD?

Comment (christinahedges) [reply lightkurve/correctors.py]: It says TIME_MJD in the CBV file and it has values  ~55000 so I thought it was MJD

**Comment by christinahedges** [comment]
screen shot 2018-02-27 at 10 29 59 am

**Comment by barentsen** [comment]
Oh, ok, let's merge!

**Pull request merged**